test/jdk/tools/jpackage/share/AddLauncherTest.java

```diff
@@ -200,6 +202,33 @@ public void testMainLauncherIsModular(boolean mainLauncherIsModular) {
         .applyTo(cmd);
 
         cmd.executeAndAssertHelloAppImageCreated();
+
+        // verify using each additional launchers cfg file that it has the
+        // main-jar/class or module/class
+
+        CfgFile cfg = cmd.readLauncherCfgFile("ModularAppLauncher");
+        String moduleValue = cfg.getValue("Application", "app.mainmodule");
+        String mainClass = null;
+        String classpath = null;
+        TKit.assertEquals(moduleValue, JavaAppDesc.parse(
+                modularAppDesc.toString()).setBundleFileName(null).toString(),
+                "app.mainmodule value in cfg file not as expected");
+
+        TKit.trace("moduleValue: " + moduleValue + " mainClass: " + mainClass
+                    + " classpath: " + classpath);
```

Comment:
`classpath` variable is `null` at this point. What is the point to log its value? The whole log statement is excessive because `classpath` variable is `null` and value of `moduleValue` variable is logged implicitly in TKit.assertEquals() call.

Reply:
ok - implemented as you suggested